tests/test_election.py

```diff
@@ -127,3 +127,17 @@ def test_event_type_enum():
     assert TimetableEvent.VAC_APPLICATION_DEADLINE.value == "VAC application deadline"
 
 
+def test_is_before():
+    election = from_election_id("parl.2019-02-21", country=Country.ENGLAND)
+    assert election.is_before(TimetableEvent.REGISTRATION_DEADLINE) == True
+    assert election.is_before(TimetableEvent.SOPN_PUBLISH_DATE) == True
+    assert election.is_before(TimetableEvent.POSTAL_VOTE_APPLICATION_DEADLINE) == True
+    assert election.is_before(TimetableEvent.VAC_APPLICATION_DEADLINE) == True
+
+
+def test_is_after():
+    election = from_election_id("parl.2019-02-21", country=Country.ENGLAND)
+    assert election.is_after(TimetableEvent.REGISTRATION_DEADLINE) == False
+    assert election.is_after(TimetableEvent.SOPN_PUBLISH_DATE) == False
+    assert election.is_after(TimetableEvent.POSTAL_VOTE_APPLICATION_DEADLINE) == False
+    assert election.is_after(TimetableEvent.VAC_APPLICATION_DEADLINE) == False
```

Comment:
These tests suggest that the logic is backwards? `is_after` should be `True` for everything from 2019?

Reply:
This is the case with the changes suggested above.